test/sampletests/withverify/verifyWithinPerform.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  before(client) {
+    client.globals.calls++;
+  },
+  
+  demoTest(client) {
+    client.url('http://localhost')
+      .verify.equal(0, 1, 'custom message')
```

Comment:
are the custom messages necessary

Reply:
copied it from somewhere, does not harm 👻